### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by SinaRahimii75** [HoseinRezaeeM/DigiWallet_functional-interface#14] [General]
import java.util.*;
class Human{
int kindness;
int money;
public Human(int kindness, int money) {
this.kindness = kindness;
this.money = money;
}
}
public class Containers {
public static void main(String[] args) {
List<Human> list = Arrays.asList(new Human(1, 10), new Human(5, 2));
Comparator<Human> comparator = new Comparator<Human>(){
@Override
public int compare(Human o1, Human o2) {
return o1.kindness<o2.kindness?-1:o1.kindness==o2.kindness?0:+1;
}
};
System.out.println(Collections.max(list, comparator).money);
}
}

**Comment by HoseinRezaeeM** [comment]
Answer is 2 
implement Comparator functionalinterface annoymousClass
and implement compare method

**Comment by SinaRahimii75** [comment]
thanks
